instagram_web_api/client.py

```diff
@@ -871,9 +872,25 @@ def post_photo(self, photo_data, caption=''):
             headers['Content-Type'] = 'application/x-www-form-urlencoded'
             del headers['Content-Length']
             endpoint = 'https://www.instagram.com/create/configure/'
+
+            if user_tags:
+                user_tags_ = {"in": []}
+                cryptogen = random.SystemRandom()
+                if isinstance(user_tags, list):
+                    if len(user_tags) > 60:
+                        warnings.warn('Adding more than 60 users will put your account at banned risk', UserWarning)
+                    for user_id in user_tags:
+                        user_tags_["in"].append({"user_id": user_id, "position": [cryptogen.random(), cryptogen.random()]})
```

Comment:
Forcibly putting user tags at random positions is not a good idea at all imo.

Reply:
maybe adding an array with the x and y positions of each tag? i like the tagging idea, but i agree with you, adding it at random is very bad.